backend/routes/api/quickstarts/quickStartUtils.js

```diff
@@ -2,6 +2,9 @@ const createError = require('http-errors');
 const constants = require('../../../utils/constants');
 const getMockQuickStarts = require('../../../__mocks__/mock-quickstarts/getMockQuickStarts');
 
+// TODO: Don't use local quickstarts
+const USE_LOCAL_QUICK_STARTS = true;
```

Comment:
required: Might be good to more fully explain/annotate as to why this is hardcoded and the following conditions always fire... the comment currently feels... ironic?